WooCommerce/Classes/POS/Presentation/ItemRowView.swift

```diff
@@ -59,11 +49,34 @@ struct ItemRowView: View {
         .padding(.vertical, Constants.verticalPadding)
         .shadow(color: Color.black.opacity(0.08), radius: 4, y: 2)
     }
+
+    @ViewBuilder
+    private var productImage: some View {
+        if dynamicTypeSize >= .accessibility3 {
+            EmptyView()
```

Comment:
This feels like a good choice to me, at certain point of accessibility it seems sensible to maximize the available space by showing only the critical information (name/price) and hide other components that are "not that important". Perhaps is something we should do more often.